### ISSUE TITLE
Final smooth-over on Kedro Academy decks following first training session

### ISSUE THREAD

**Issue by stichbury** [kedro-org/kedro-devrel#34]
We need to take another look at the examples on the decks, and take any other feedback into account, to finalise the first 4 modules.

**Comment by astrojuanlu** [comment]
One important thing to consider is whether to optimize these slides for reading material or for screensharing material. One of the feedback points was that when the slides had too much content they became a distraction from the instructor explanations.

With that in mind, maybe we could organize the slides in such a way that we could hide the denser ones during the screensharing (resorting to live coding and exercises instead) and keep them in the final PDF delivered to the participants.

**Comment by stichbury** [comment]
I have been through the decks again today, mindful that we've changed a few things since the last course:

* replaced Replit with Gitpod --> I removed all mention of Replit
* split into 2 sections of 3 hours --> I removed the slide with timings for the 4 hour course. I am sure you have your own timings but I'm not sure it's necessary to share with the students
* swapped dependency management and python functions --> I swapped how these were presented in the introduction but haven't renumbered the course sections in Box until we are sure we are using this in future iterations
* adding IDE usage --> I have not done anything with this (it's a separate ticket)

In summary: I made a very few adjustments to the decks and they're ready for review by you now, @astrojuanlu. I would suggest that you may want to update the introduction slide that says what the students will learn, **to say which day each section occurs on** (I didn't do this as I wasn't sure how you were splitting the modules over the days). 

Note: I haven't hidden any slides from view (as per your suggestion above). I think it's a good idea when you've decided which slides you're going to talk about in a demo rather than talk over in the presentation. You're better placed to do that as part of the review, so please can I leave you to add the Hide options?

**Comment by astrojuanlu** [comment]
Consider the decks reviewed ✔️ I have also organized the outline to divide the sections accordingly. Might make small adjustments but for now we can close this ticket.
